packages/atomic/src/components/atomic-sort-criteria/atomic-sort-criteria.tsx

```diff
@@ -9,12 +10,14 @@ import {Component, Prop, Element} from '@stencil/core';
   shadow: false,
 })
 export class AtomicSortCriteria {
```

Comment:
Unrelated to the PR changes, but it would be cleaner/more grammatically correct if we used the singular here, either `AtomicSortCriterion` or `AtomicSortExpression` (I prefer the latter).

Criteria suggests "multiple things". While multiple criteria can be chained together separated by delimiters, it is ultimately one expression.

Not suggesting to do this now. If we rename, let's use a separate PR,

Reply:
Criteria can be one or multiple, right?
https://www.merriam-webster.com/words-at-play/criteria-vs-criterion-singular-plural-grammar
>  evidence shows that criteria is frequently being used as a singular as well as a plural

Here a user can input one or multiple criteria
- date descending, size ascending
- relevancy

Reply:
Personally, I haven't seen the singular trend like with other words such as `data`. I found the last paragraph in that article summarized the current state well as emerging:

> Unlike data and agenda, however, criteria still shows use as a count noun (“here are three criteria”) alongside the emerging singular. Only time will tell if criteria will continue to show preference as the singular form over criterion.

On the second point, I see the multiple criteria as forming one criterion expression.

Anyway, I found the plural name strange in this context. I wanted to point this out in case anyone else was thinking the same, considering it would be more difficult to change after the official release.

Reply:
Given that we've used `criterion` in some places to mean singular, I would also vote for using `criterion` here.